charms/candid/requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
```

Comment:
Didn't know you could do this! Maybe update tox to not pull other requirements.txt now